.github/workflows/validate_charts.yml

```diff
@@ -0,0 +1,29 @@
+name: validate charts
```

Comment:
The validate-charts and pr-build workflow are having the same trigger condition. Can we move this job as part of pr-build.yml workflow?